homeassistant/util/volume.py

```diff
@@ -45,8 +59,12 @@ def convert(volume: float, from_unit: str, to_unit: str) -> float:
 
     result: float = volume
     if from_unit == VOLUME_LITERS and to_unit == VOLUME_GALLONS:
-        result = __liter_to_gallon(volume)
+        result = liter_to_gallon(volume)
     elif from_unit == VOLUME_GALLONS and to_unit == VOLUME_LITERS:
-        result = __gallon_to_liter(volume)
+        result = gallon_to_liter(volume)
+    elif from_unit == VOLUME_CUBIC_METERS and to_unit == VOLUME_CUBIC_FEET:
+        result = cubic_meter_to_cubic_feet(volume)
+    elif from_unit == VOLUME_CUBIC_FEET and to_unit == VOLUME_CUBIC_METERS:
+        result = cubic_feet_to_cubic_meter(volume)
```

Comment:
We should add a couple of tests in the volume util test module similar to this test:
https://github.com/home-assistant/core/blob/8cb3a485e07374c538c593aa94388549dd149e59/tests/util/test_volume.py#L46-L49